src/middleware/error.js

```diff
@@ -38,6 +38,13 @@ class MiddlewareError extends idrErr.IDRError {
 
     return err
   }
+
+  recordTooLarge () {
```

Comment:
It seems like this function isn't getting used to set the error message for [largeInputErrorHandler](https://github.com/CVEProject/cve-services/pull/466/files#diff-4bb01e986b6f28839e7b47ff5b4e002c4e211378418d604a7cb12570592d68faR206).
We should make sure the endpoint test checks the contents of the error response so we can know what message is getting sent back.